packages/quantic/force-app/main/default/lwc/quanticFacet/quanticFacet.html

```diff
@@ -16,41 +19,77 @@
         >
         </lightning-button-icon>
         <template if:true={hasActiveValues}>
-          <button class="facet__clear-filter slds-button slds-grid slds-grid_vertical-align-center slds-p-top_x-small slds-p-horizontal_x-small" onclick={clearSelections}>
-            <lightning-icon class="slds-current-color slds-m-right_xx-small" icon-name="utility:close" alternative-text={altText} size="xx-small"></lightning-icon>
-            <span class="nowrap pill__text-container slds-truncate">{clearFilterLabel}</span>
+          <button
+            class="facet__clear-filter slds-button slds-grid slds-grid_vertical-align-center slds-p-top_x-small slds-p-horizontal_x-small"
+            onclick={clearSelections}
+          >
+            <lightning-icon
+              class="slds-current-color slds-m-right_xx-small"
+              icon-name="utility:close"
+              alternative-text={altText}
```

Comment:
Since the parent is a button, I would instead give thi icon `aria-hidden`.

Reply:
+1 for this, you already have text on the button's span, no need to have a label on the icon, and aria-hidden will make screen readers completely ignore it.